crates/core/tedge_agent/src/tedge_to_te_converter/error.rs

```diff
@@ -0,0 +1,27 @@
+use tedge_actors::RuntimeError;
+use tedge_api::alarm::ThinEdgeJsonDeserializerError;
+
+#[derive(Debug, thiserror::Error)]
+#[allow(clippy::enum_variant_names)]
+pub enum TedgetoTeConverterError {
```

Comment:
This error type can be removed.

Reply:
You are correct: inserting the alarm severity in the message might lead to errors.

Two cases are unused though:
- `FromChannelError`
- `AlarmError`